File: 1672/JavaScript/One.js
var maximumWealth = function (accounts) {
  let sum = 0,
    max = Number.MIN_VALUE;
  for (let i = 0; i < accounts.length; i++) {
    sum = 0;
    for (let j = 0; j < accounts[0].length; j++) {
      sum += accounts[i][j];
    }
    max = Math.max(max, sum);
  }
  return max;
};

File: 1672/CPP/One.cpp
#include <iostream>
#include <vector>

using namespace std;

int maximumWealth(vector<vector<int>> &accounts)
{
    int max_val = INT_MIN, sum = 0;
    for (auto i : accounts)
    {
        sum = 0;
        for (auto j : i)
        {
            sum += j;
        }
        max_val = max(max_val, sum);
    }
    return max_val;
}

int main()
{
    vector<vector<int>> accounts = {{1, 2, 3}, {3, 2, 1}};
    cout << maximumWealth(accounts) << endl;
    return 0;
}